src/insanely_fast_whisper/cli.py

```diff
@@ -222,7 +222,13 @@ def post_process_segments_and_transcripts(new_segments, transcript, group_by_spe
     type=str,
     help="Provide a hf.co/settings/token for Pyannote.audio to diarise the audio clips",
 )
-
+parser.add_argument(
+    "--diarization_model",
+    required=False,
+    default="pyannote/speaker-diarization",
```

Comment:
```suggestion
    default="pyannote/speaker-diarization-3.1",
```
I think 3.1, as default, makes more sense. (plus fully PyTorch)